spec/system/admin/bulk_order_cancellation_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+describe '
+  As an Administrator
+  I want to be able to delete orders in bulk
+' do
+  include AdminHelper
+  include AuthenticationHelper
+  include WebHelper
+
+  context "deleting orders" do
+    let!(:o1) {
+      create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                      completed_at: Time.zone.now )
+    }
+    let!(:o2) {
+      create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                      completed_at: Time.zone.now )
+    }
+
+    before :each do
+      login_as_admin_and_visit spree.admin_orders_path
+    end
+
+    it "deletes orders" do
+      # Verify that the orders have a STATE of COMPLETE
+      save_screenshot "~/order.png"
+      page.assert_selector('span', text: 'COMPLETE', count: 2)
```

Comment:
Ah, cool, that's much better. It doesn't really matter but we usually use the expect syntax:
```suggestion
      expect(page).to have_selector('span', text: 'COMPLETE', count: 2)
```

Reply:
I changed it like you said just for consistency sake. 

Also I will keep in mind your comment below. I have not been adding commits for my changes because I was trying to keep a clean git history but I can see how that is a bother when you have to review and can't immediately see what's changed. Sorry for the trouble!
I'll add commits from now on requested changes and sqash them when it makes sense once review's finished.